.changeset/grumpy-socks-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/cdk": patch
```

Comment:
```suggestion
"@guardian/cdk": minor
```

Looking at the [release history](https://github.com/guardian/cdk/releases), I think we'd normally use `minor` for feature releases (like this) and `patch` for dependency upgrades etc.

Reply:
updated this to minor release.